components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -80,14 +81,15 @@ public ServiceURL build() throws URLBuilderException {
 
         String protocol = fetchProtocol();
         String hostName = fetchHostName();
```

Comment:
Also this one as fetchProxyHostName

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2949/commits/505c8f4e1f9f23a167ac4bac3b67bac5bab186cc